### ISSUE TITLE
GDALException at {api_endpoint} OGR failure.

### ISSUE THREAD

**Issue by utkuturunc** [dulaccc/heroku-buildpack-geodjango#8]
Hi,

I am using rest framework with gis extension. When a geo endpoint is accessed this error shows up:

```
GDALException at /api/v1/geo/addresses/
OGR failure.
```
and this line seems to be triggering the error:
return json.loads(GEOSGeometry(value).geojson) 

Same code works fine on my local machine.

**Comment by dulaccc** [comment]
Are you using the `1.0` version of the buildpack ?

**Comment by utkuturunc** [comment]
Yes. Sorry forgot to mention that.

**Comment by dulaccc** [comment]
No problem, alright I'm looking into that right now !

**Comment by dulaccc** [comment]
On a clean install I did

```sh
$ heroku run python manage.py shell
>>> from django.contrib.gis.geos import GEOSGeometry
>>> GEOSGeometry('POINT(5 23)').geojson
u'{ "type": "Point", "coordinates": [ 5.0, 23.0 ] }'
```

No error for me... can you try to reproduce the above steps in a django shell ? to see if the error is triggered ?

**Comment by utkuturunc** [comment]
That does not trigger any errors.

Here is the link to my api endpoint:
http://bn-backend.utkuturunc.com/api/v1/geo/addresses/

This works for me using development server in my local machine. 

By the way, I tried cleaning cache, resetting heroku repo and rebuilding.

**Comment by dulaccc** [comment]
Thanks for anticipating my next question on the cache, resettings and rebuild thing ;)
When I try your api endpoint it doesn't fail... or maybe a soft fail only you can see in the logs ?

**Comment by utkuturunc** [comment]
You were probably getting an authentication error. It should now fail.

**Comment by utkuturunc** [comment]
Here is the logs from heroku. Should be more helpful.

```
←[32m2015-06-21T21:37:34.337413+00:00 heroku[api]:←[0m Starting process with command `bash` by utkuturunc93@gmail.com
←[35m2015-06-21T21:37:52.028374+00:00 heroku[run.8092]:←[0m Awaiting client
←[35m2015-06-21T21:37:52.079190+00:00 heroku[run.8092]:←[0m Starting process with command `bash`
←[35m2015-06-21T21:37:52.133241+00:00 heroku[run.8092]:←[0m State changed from starting to up
←[35m2015-06-21T21:44:13.723077+00:00 heroku[run.8092]:←[0m Process exited with status 0
←[35m2015-06-21T21:44:13.739642+00:00 heroku[run.8092]:←[0m State changed from up to complete
←[36m2015-06-21T21:44:41.633481+00:00 app[web.1]:←[0m GDAL_ERROR 4: Unable to open EPSG support file gcs.csv.
←[36m2015-06-21T21:44:41.633485+00:00 app[web.1]:←[0m Try setting the GDAL_DATA environment variable to point to the
←[36m2015-06-21T21:44:41.633488+00:00 app[web.1]:←[0m directory containing EPSG csv files.
←[33m2015-06-21T21:44:43.291856+00:00 heroku[router]:←[0m at=info method=GET path="/api/v1/geo/addresses/" host=bn-backe
nd.utkuturunc.com request_id=045b82ed-585b-48bb-ae0a-754904deedeb fwd="92.45.212.159" dyno=web.1 connect=2ms service=277
7ms status=500 bytes=348543
←[36m2015-06-21T21:44:43.289179+00:00 app[web.1]:←[0m {address space usage: 368730112 bytes/351MB} {rss usage: 51732480
bytes/49MB} [pid: 8|app: 0|req: 9/33] 10.69.24.23 () {50 vars in 1018 bytes} [Sun Jun 21 21:44:40 2015] GET /api/v1/geo/
addresses/ => generated 348351 bytes in 2772 msecs (HTTP/1.1 500) 4 headers in 192 bytes (4 switches on core 0)
```

**Comment by dulaccc** [comment]
I've integrated your fix into the 3631eb0 (tagged `1.1`)
Thanks for your findings !
